### DISCUSSION TITLE
Image dataUrl support?

### DISCUSSION THREAD

**Discussion by jianghai** [Brooooooklyn/canvas#673] [Ideas]
```js
const img = new Image();
img.src = <Buffer>
```
I have image data which is dataUrl，Though dataUrl could be parsed to buffer `Buffer.from(dataUrl.split(',')[1], 'base64');` but brings a little performance loss.

**Comment by twlite** [comment]
Data uri is supported via `loadImage` function 

```js
import { loadImage } from '@napi-rs/canvas'

const image = await loadImage('dataUri')
```

You can also find the test for it here

https://github.com/Brooooooklyn/canvas/blob/171a20ec3fd9325ae2e70aaf1a84bb0ee7f5fe12/__test__/loadimage.spec.ts#L33-L38
